enhancements/ibm-public-cloud/ibm-public-cloud-support.md

```diff
@@ -0,0 +1,251 @@
+
+#### Cluster Version Operator Changes
+In phase I the cluster version operator excludes operator manifests that
+have an annotation in the form:
+
+`exclude.release.openshift.io/[identifier]=true`
+
+whenever a global variable in the form
+
+`EXCLUDE_MANIFESTS=[identifier]`
+
+is present. For IBM public cloud, the value of \[identifier\] is 
+`internal-openshift-hosted`
+
+#### Excluded Manifests
+Manifests for the components that should be skipped by the CVO whenever
+the `EXCLUDE_MANIFESTS=internal-openshift-hosted` environment variable is 
+present, should include the annotation to exclude them. In most cases, only the
+minimum set of manifests to allow skipping the component should be annotated.
+However, in the case of the Machine API and Machine Configuration operators, 
+the CRDs that represent machines, machinesets and autoscalers should also be
+skipped. Monitoring alerts for components that do not get installed in the user
+cluster should also be skipped where possible.
+
+#### Console Changes
+The console should not report the control plane as being down if no metrics
+datapoints exist for control plane components in this configuration.
+
+### GA
+
+For GA, there is more native support for IBM Public Cloud as a supported
+cloud provider in the product. Operators for control plane
+components (based on the control plane self-hosted operators) are deployed on 
+the management cluster.
+
+Components and changes required for this phase:
+
+#### Control Plane Operators
+Break up the single Beta control plane operator into different operators that
+vendor code from the self-hosted control plane operators in the product. The vendored
+code should include config observers that assemble a new configuration for their
+respective control plane components. This will ensure that drift in future versions
+is kept under control and that a single code base is used to manage control plane
+configuration.
+
+#### Cluster Version Operator Changes
+The CVO should support the concept of an install profile that allows selecting manifests
+based on which profile is in effect. This should make it possible to modify the node
+selector of SLOs that are installed on IBM Public Cloud clusters to target worker nodes.
+(See separate proposal for installation profiles).
+
+#### Ingress Operator
+Add native support for IBM Public Cloud as a cloud provider.
+
+#### Storage Operator
+Create storage classes needed for IBM Public Cloud.
+
+#### Image Registry Operator
+Support IBM Public Cloud natively by provisioning the appropriate storage (RWO PVC)
+
+#### OAuth Server
+Support IBM public cloud as a provider.
+
+### Post-GA
+
+Add support for managed RHCOS nodes.
+
+#### Managed Workers
+RHCOS adds support for bootstrapping on IBM Public Cloud. The MCO is added to 
+the components that get installed on the management cluster.  This enables upgrading 
+of RHCOS nodes using the same mechanisms as in self-hosted OpenShift.
```

Comment:
So compute nodes still run machine-config daemons and cluster admins can write MachineConfig entries, create MachineSets, and all that good stuff?  They just don't have any objects representing or control over the control-plane machines?